### ISSUE TITLE
3.0 - Update tabbed settings navigation

### ISSUE THREAD

**Issue by robincornett** [easydigitaldownloads/easy-digital-downloads#8055]
### Explain your enhancement (please be detailed)
We should update our settings pages' tabbed navigation to be consistent with [WordPress Core](https://core.trac.wordpress.org/ticket/43398#comment:17).

### Justification or use case
Summary of what Core changed:

> Accessibility: Improve the semantics of "tabs" that are not real ARIA tabs.
>
> Some "visual tabs" in the WordPress admin used links wrapped in a `h2` heading. While these tabs look like ARIA tabs, they're just navigation menus and should be marked-up as such to be correctly exposed to assistive technologies.
>
> -  changes the wrapping `<h2>` to a `<nav>` element: worth reminding `<nav>` elements also define ARIA landmarks
> -  adds an `aria-label` to the `<nav>` elements so they can be distinguished from other `<nav>` elements in the page
> -  adjusts the headings level in the Credits page

It's worth noting that updating this improves markup and accessibility and should be backwards compatible with 4.x versions of WordPress, as the linked changeset doesn't modify Core CSS, except for the "About" page. Ran into this myself when trying to work on #8040, as that is related.
